src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -695,6 +711,52 @@ private void determineUnitsToAttackWith(final List<ProTerritory> prioritizedTerr
       Map<Unit, Set<Territory>> sortedUnitAttackOptions =
           tryToAttackTerritories(prioritizedTerritories, alreadyMovedUnits);
 
+      // Clear bombers
```

Comment:
So ProTerritoryManager is essentially that object. But its already become too bloated and needs broken up.